### PR TITLE
55 create a pages folder

### LINKED ISSUE #55: Create a Pages Folder

### PR DESCRIPTION
**Pull request by sraban-k-jena**
- [x] Add a new Pages Folder

### CODE REVIEW

**Review by sraban-k-jena — COMMENTED** [review]
- [x] Add new Features in Admin and Customer Folder

File: frontend/src/App.jsx
Comment (smruti-banty) [inline frontend/src/App.jsx:1]: change root routes

**Comment by sraban-k-jena** [comment]
- [x] Change Roue Routes

**Review by smruti-banty — APPROVED** [review]
LGTM.

**Pull request merged**